notices.json

```diff
@@ -34,17 +34,17 @@
     }
   },
   {
-    "id": "desktop_upgrade_v5.1",
+    "id": "desktop_upgrade_v5.2",
     "conditions": {
-      "audience": "all",
+      "audience": "sysadmin",
```

Comment:
@amyblais Shouldn't all users be getting this notice?

Reply:
Neil mentioned here that it would be targeted for sys admins, but I may have misunderstood https://community.mattermost.com/core/pl/p9m9idso8fd75gr5o3f3zp87xe. @nab-77 let me know if this is for sys admins or all users.

Reply:
This is the desktop notice so should be for all.